Southbank-Centre/southbankcentre.org
# 104715252 prevent unnecessary tests

@djvern to tests:

- Log into CI and see that this branch hasn't build automatically
- Install the branch locally and serve the site
- Modify a functional test so that it fails
- Run the functional tests and see that the test fails, and the test runner exits immediately after the failure

Also included in this PR is a fix for ngconstant when serving the site locally
